simpl/threadlocal.py

```diff
@@ -0,0 +1,52 @@
+"""Threadlocal."""
+import collections
+import threading
+
+THREAD_STORE = threading.local()
+DEFAULT_NAMESPACE = 'call_context'
+
+
+class ThreadLocalDict(collections.MutableMapping):
+
+    """A dict whose data is local to the thread."""
+
+    def __init__(self, varname, *args, **kwargs):
+        self.varname = varname
+        self.args = args
+        self.kwargs = kwargs
+
+    def __repr__(self):
+
+        under = repr(self._get_local_dict())
```

Comment:
Curiosity killed the cat, I know, but I can't help but ask... why is the variable called `under`?